index.js

```diff
@@ -49,7 +49,7 @@ module.exports = {
             resolve()
           ]
         },
-        nodeModulesPath: this.nodeModulesPath,
+        _nodeModulesPath: this._nodeModulesPath,
```

Comment:
This should be `nodeModulesPath: this._nodeModulesPath`. It is a parameter for Broccoli Rollup and they decide what it's called.

Reply:
Good catch. :+1: Sometimes a find and replace isn't the safest way to do it. :smile: